dataprep/data_connector/schema.json

```diff
@@ -145,6 +148,9 @@
                     {
                         "type": "boolean"
                     },
+                    {
+                        "type": "integer"
+                    },
                     {
                         "type": "object",
                         "required": [
```

Comment:
The table configs should be a separate PR to DataConnectorConfigs repo. 
Also, please squash the PR and rebase to the `develop` branch.

Thanks!